File: python/kasirmini.py
nama = input("Masukkan nama barang: ")
harga = float(input("Masukkan harga barang: "))
jumlah = int(input("Masukkan jumlah beli: "))

total = harga * jumlah

if total > 26500:
    diskon = total * 0.06
else:
    diskon = 0

total_bayar = total - diskon

print("\n===== STRUK PEMBELIAN ======")
print(f"Nama Barang    : {nama}")
print(f"Harga Satuan    : {harga}")
print(f"Jumlah Beli     : {jumlah}")
print(f"Total Harga     : {total}")
print(f"Diskon    : {diskon}")
print(f"Total Bayar    : {total_bayar}")
print("=================================")

File: java/kasirmini.java
import java.util.Scanner;

public class kasirmini {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Masukkan nama barang: ");
        String nama = input.nextLine();

        System.out.print("Masukkan harga barang: ");
        double harga = input.nextDouble();

        System.out.print("Masukkan jumlah beli: ");
        int jumlah = input.nextInt();

        double total = harga * jumlah;
        double diskon;

        if (total > 26500) {
            diskon = total * 0.60;
        } else {
            diskon = 0;
        }

        double totalBayar = total - diskon;

        System.out.println("\n===== STRUK PEMBELIAN =====");
        System.out.println("Nama Barang   : " + nama);
        System.out.println("Harga Satuan  : " + harga);
        System.out.println("Jumlah Beli   : " + jumlah);
        System.out.println("Total Harga   : " + total);
        System.out.println("Diskon        : " + diskon);
        System.out.println("Total Bayar   : " + totalBayar);
        System.out.println("============================");
    }
}

